### PR TITLE
Decompile `no0` Ouija Table components sub-entity

### PR DESCRIPTION
**Pull request by JoshSchreuder**
Also split into a separate entity file for Ouija Table

Matching both
PSX: https://decomp.me/scratch/oph7c
PSP: https://decomp.me/scratch/MTVot

I'll probably extract the data on this one shortly just to round it out

### CODE REVIEW

File: src/st/no0/e_ouija_table.c [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+            CreateEntityFromEntity(E_OUIJA_TABLE_COMPONENT, self, otherEntity);
+            otherEntity->params = 0;
+            otherEntity->ext.ouijaTableContents.parent = self;
+            otherEntity->posX.i.hi -= 4;
+            otherEntity->posY.i.hi -= 16;
+
+            otherEntity = BEAKER;
+            CreateEntityFromEntity(E_OUIJA_TABLE_COMPONENT, self, otherEntity);
+            otherEntity->params = 1;
+            otherEntity->ext.ouijaTableContents.parent = self;
```
Comment (sozud) [inline src/st/no0/e_ouija_table.c:83]: I didn't check but you're going to want to have a static_assert on the pointer and an item afterwards to make sure it all aligns

**Review by sozud — APPROVED** [review]

File: src/st/no0/e_ouija_table.c (hunk 1 above)
Comment (JoshSchreuder) [reply src/st/no0/e_ouija_table.c:83]: Sorry not sure what you mean by an item afterwards, but I added a static assert, hopefully that's right

Comment (JoshSchreuder) [reply src/st/no0/e_ouija_table.c:83]: Failing on the i386 build now. I'm not sure how to fix sorry, if someone can run me through what I need to change that would be great

File: include/entity.h [hunk 2]
```diff
@@ -1997,11 +1997,16 @@ typedef struct {
 } ET_OuijaTable;
 
 typedef struct {
+#ifdef PLATFORM_64BIT
+    s32 _align_parent[2];
+#endif
```
[the remark this answers is not in the document]
Comment (JoshSchreuder) [reply include/entity.h]: Reverted this as discussed

**Pull request merged**